acc_test/resource_cluster_test.go

```diff
@@ -46,6 +43,21 @@ func TestAccCluster(t *testing.T) {
 	})
 }
 
+func TestAccCluster(t *testing.T) {
```

Comment:
[desired] I think this test should be split into 2. One single dc, one multi dc

Reply:
cool!